style.css

```diff
@@ -114,7 +191,7 @@ body {
 .myRecentWork {
   margin: 22px;
   display: grid;
-  gap: 24px;
+  gap: 12px;
   justify-items: center;
 }
 
```

Comment:
[OPTIONAL]Great job @PierreBarba 😊. Kindly give your card containers padding to create space around the containers and also to meet up with the Figma design.

Also, kindly make the `post` cards take the full width of their mother container in accordance with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=jv56JbMIOvKnWOOj-0) design.

| Your Design |Figma Design |
| :---:   | :---: | 
| <img width="1470" alt="image" src="https://user-images.githubusercontent.com/73854999/204032252-78037cd0-4737-4926-a370-2204333d18ab.png">|<img width="725" alt="image" src="https://user-images.githubusercontent.com/73854999/204032350-f817ea58-b8e8-4078-8002-7c88e6cd6e6b.png">|

Reply:
Thanks for the optional change.